src/controllers/bookController.ts

```diff
@@ -25,6 +28,39 @@ class BookController {
             error_description: 'Endpoint not implemented yet.',
         });
     }
+
+    getAllBooks(req: Request, res: Response) {
```

Comment:
I know this is what you're working on, but it would be good to commonise some of this stuff so that you can reuse it for future endpoints